lib/generic/cluster.h

```diff
@@ -15,6 +15,9 @@ namespace MouseTrack {
 /// The corresponding point cloud is not linked in this data structure.
 class Cluster {
 public:
+    /// Construct empty cluster object
+    Cluster();
```

Comment:
Thanks, I forgot this one. :)

Reply:
Is this necessary?

Reply:
Yes, for example if you want to create an empty cluster object and fill it with points afterwards, or if you initialize a std::vector of clusters it calls the empty constructor on every element.

Reply:
:+1: